Contributing.md

```diff
@@ -0,0 +1,120 @@
+# PlanetaryPy Community Contributing Guide
+
+This document describes a very simple process suitable for activities
+in the Planetary Software Organization ecosystem, including the TC
+itself.
+
+The goal of this document is to create a contribution process that:
+
+* Encourages new contributions.
+* Encourages contributors to remain involved.
+* Avoids unnecessary processes and bureaucracy whenever possible.
+* Creates a transparent decision making process which makes it clear how
+contributors can be involved in decision making.
+
+
+## Vocabulary
+
+* A **Contributor** is any individual creating or commenting on an
+	issue or pull request.
+* A **Committer** is a subset of contributors who have been given
+	write access to the repository.
```

Comment:
I think we need to be more clear here. We will have more than one repository? Unless we want to write PlanetaryPy GH Org? Not sure...

Reply:
So this is the `Contributing` document for the TC repo. The planetarypy repo and any affiliated package repos will also need their own `Contributing` documents that will probably be derived from this one in some way, but perhaps altered.

Reply:
Oh, gotcha.